python/sidecar/ingest.py

```diff
@@ -203,6 +222,7 @@ def _create_references_for_grobid_failures(self) -> list[Reference]:
             references.append(
                 Reference(
                     source_filename=source_pdf,
+                    status=typing.IngestStatus.FAILURE,
```

Comment:
I'm not convinced we should call this an `failure`. Just because Grobid was unable to process it does not mean we cannot do things with it. See [this issue for example](https://github.com/refstudio/refstudio/issues/175).

Reply:
I agree that for now "failure" is the most appropriate since we are only doing Grobid.